src/tools/haxelib/Data.hx

```diff
@@ -38,6 +38,7 @@ typedef UserInfos = {
 typedef VersionInfos = {
 	var date : String;
 	var name : String;
+	var downloads : Int;
```

Comment:
Any thoughts on this @ncannasse ?

We could also create a different typedef for use on the website that doesn't affect the remoting API.